kamon-core/src/main/scala/kamon/trace/Span.scala

```diff
@@ -174,14 +197,14 @@ object Span {
   val BooleanTagTrueValue = "1"
```

Comment:
This is used only in one place a few lines up, so I'm not sure what's the point of having it here? Also the `"1"` literal is used in `SpanContextCodec`, so we should probably put it somewhere publicly accessible, and use it consistently.